core/components/_helpers/avatar-url.js

```diff
@@ -9,6 +9,9 @@ const urlTemplate = (hash, initials, size) =>
   `https://s.gravatar.com/avatar/${hash}?s=${size}&r=pg&d=${initialsAvatar(initials)}`
 
 export default function avatarUrl(email, initials, size = 480) {
-  const hash = md5(email.toLowerCase())
+  if (!email && !initials)
+    throw new Error('You need to provide either `email`, `initials` or both to the Avatar.')
```

Comment:
isn't that why we do have a default profile svg?

Reply:
That case is handled by the Avatar component itself (by rendering a local SVG, so not a URL).

Reply:
Nope, just checked. This is a CDN URL, let me change this so we can centralize it :)

Reply:
Done